composables/helpers/buildPaymentMethodInput.ts

```diff
@@ -0,0 +1,39 @@
+import type { BillingInfo, PaymentActionInput } from "@/server/types/GraphQL"
+import { creditCardPaymentGetters } from "@/lib/getters/creditCardPaymentGetters"
+
+export const buildPaymentMethodInput = (
+  paymentType,
+  currencyCode,
+  paymentWorkflow,
+  checkout,
```

Comment:
Pass in the total to apply instead of the whole "checkout" object

Reply:
checkoutGetter.getTotal() accept "checkout" obj. Other getters are also accepting checkout obj instead specific one. Should be keep it as is for consistency or we can change it?